android/src/main/kotlin/com/hypertrack/sdk/flutter/common/Result.kt

```diff
@@ -0,0 +1,48 @@
+package com.hypertrack.sdk.flutter.common
+
+internal sealed class Result<S> {
```

Comment:
Shouldn't it be internal sealed class Result<S, T>?
Maybe also T -> R to be
S like Success
R like Result?

Reply:
I am not sure what do you mean, by we don't have any failures except Exception